### PR TITLE
📝 Add External Link: Instrument a FastAPI service adding tracing with OpenTelemetry and send/show traces in Grafana Tempo

### PR DESCRIPTION
**Pull request by softwarebloat**

### CODE REVIEW

File: docs/en/data/external_links.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 articles:
   english:
+  - author: Nicoló Lino
+    author_link: https://www.nlino.com
+    link: https://github.com/softwarebloat/python-tracing-demo
+    title: 'Instrument a FastAPI service adding tracing with OpnTelemetry and send/show traces in Grafana Tempo'
```
Comment (willingc) [inline docs/en/data/external_links.yml:6]:
```suggestion
    title: 'Instrument a FastAPI service adding tracing with OpenTelemetry and send/show traces in Grafana Tempo'
```

**Review by willingc — COMMENTED** [review]
Looks good. One small typo to correct. Thanks!

File: docs/en/data/external_links.yml (hunk 1 above)
Comment (softwarebloat) [reply docs/en/data/external_links.yml:6]: oh thank you very much!!

**Comment by softwarebloat** [comment]
hey @willingc , can this be merged? :)

**Comment by tiangolo** [comment]
📝 Docs preview for commit 75c0b633b2726a493741ddea70c4470951bf4db0 at: https://4b52701e.fastapitiangolo.pages.dev

**Review by hasansezertasan — APPROVED** [review]

**Comment by alejsdev** [comment]
Thank you for your contribution! @softwarebloat 🙂

And thanks for the help @willingc and @hasansezertasan ✌️

**Pull request merged**